picoquic/packet.c

```diff
@@ -77,6 +77,7 @@ int picoquic_parse_packet_header(
                 ph->offset += picoquic_parse_connection_id(bytes + ph->offset, l_srce_id, &ph->srce_cnx_id);
 
                 /* Not applicable for long packets. */
+		ph->hasspinbit = 0;
                 ph->spin = 0;
```

Comment:
I am not too picky about variable names, but I would have used "has_spin_bit" instead of "hasspinbit"